### ISSUE TITLE
Weirdly small size after boot/reboot

### ISSUE THREAD

**Issue by MoarcyVega** [Zren/plasma-applet-eventcalendar#307]
Hello, 
i have a weird issue with the event calendar-widget
The size when i click on the calendar is weirdly small after reboot or starting my computer (as shown in the screenshot):
![Screenshot_20220413_022416](https://user-images.githubusercontent.com/61259286/163080024-deb9fd9a-0aeb-4476-ba27-9c6a77bcd211.png)

After clicking on settings, the size is correct:
![Screenshot_20220413_031142](https://user-images.githubusercontent.com/61259286/163080059-b3b7826f-f29b-41fa-a51f-aeb2edd9a84d.png)

Packages are all up to date and the version of the widget is the latest one (75)
Maybe someone has the same issue or can help

**Comment by Cris70** [comment]
Same problem here, but it doesn't fix itself even after clicking on settings.
OpenSuse Tumbleweed 20220411 (KDE Plasma 5.24.4 - KDE Frameworks 5.93.0).

**Comment by Cris70** [comment]
> Same problem here, but it doesn't fix itself even after clicking on settings. OpenSuse Tumbleweed 20220411 (KDE Plasma 5.24.4 - KDE Frameworks 5.93.0).

I was able to fix it by slightly changing the sizes of all the components in the "Layout" tab and saving.
I still do not know if it will stick after a reboot.

**Comment by Peter-Ries** [comment]
Seems to have happened with update kde Framework from 5.92.xx to 5.93.xx
My sticky notes plasmoid opens only a minimini popup on click. So not only a Event calendar issue...

**Comment by Cris70** [comment]
@Peter-Ries you're right, my notes plasmoid became unusable!! And I have not yet found a workaround for that one :-(

**Comment by MoarcyVega** [comment]
> > Same problem here, but it doesn't fix itself even after clicking on settings. OpenSuse Tumbleweed 20220411 (KDE Plasma 5.24.4 - KDE Frameworks 5.93.0).
> 
> I was able to fix it by slightly changing the sizes of all the components in the "Layout" tab and saving. I still do not know if it will stick after a reboot.

It doesn't work on my pc

**Comment by lynoxy** [comment]
I am having the same problem. For now i can increase the size again with meta+right click but when i restart it gets smaller again.

**Comment by Peter-Ries** [comment]
the hint with clicking settings works - at least until I reboot. This also fixes my minimini Notes popup. After Settings both event calendar & Notes show their correct size when clicking.

I have no clue if eventcalendar "disturbs" other plasmoids or if it's a thing behind the scene that whas been "invented" with framework 5.93.xx

When I get back home I try to pay in a VM to see what's the cause and if it's a very special set of plasmoids in taskbar.

Otherwise I have no idea where to send a proper bug report (here or KDE framework guys)

**Comment by Cris70** [comment]
Unfortunately I can confirm that, after a reboot, everything is back to minuscule :-(

**Comment by Peter-Ries** [comment]
I tested in a VM running KDE Fraework 5.92 - no Event Calendar.
I added Sticky Notes to taskbar.

Update arch in my case so I had 5.93 KDE FW afterwards. Reboot.

Stiky Notes does not work anymore correctly. So KDE Framework seems to be the culprit. Eventcalendar & Sticky Notes & some more(?) suffer the same problem of not opening in correct size after click. The settings dialog of Event Calendar somehow seems to be a temporary cure doing something in KDE FW that helps other Plasmoids, too.

I'll file a bugreport @ KDE guys.

**Comment by Zren** [comment]
Seems the non duplicate bug is https://bugs.kde.org/show_bug.cgi?id=452512

Was experiencing this in TiledMenu myself (https://github.com/Zren/plasma-applet-tiledmenu/issues/125) but assume it was because my GPU was broken. Seems it's a bug where it ignores the `Layout.preferredWidth` and grabs the current `width` when the window is first displayed. Luckily EventCalendar seems to have a minimum size unlike TiledMenu.

Nate has bugged a few distros to release a patched KDE Framework 5.93. If there's still a lazy distro in a week, I could probably workaround this by setting width/height in the full representation `PopupView.qml`.

```qml
width: Layout.preferredWidth
height: Layout.preferredHeight
```

https://github.com/Zren/plasma-applet-eventcalendar/blob/master/package/contents/ui/PopupView.qml#L34-L69


You can quickly resize the popup with `Win+RightClick+Drag` (older users might need to use `Alt` instead of `Win`/`Meta) until it's fixed.

**Comment by Peter-Ries** [comment]
sorry, forgot to update here. Problem has been fixed with update of extra/plasma-framework from 5.93.0-1 to 5.93.0-2

**Comment by Zren** [comment]
https://repology.org/project/plasma-framework/versions

* [ ] Fedora 35 updates `5.91.0`
* [ ] KDE Neon `5.93.0`
    * 2022-04-21 https://old.reddit.com/r/kde/comments/u8kcsf/plasma_applet_doesnt_render_correctly_sticky_note/
* [ ] Manjaro Stable `5.92.0`
* [ ] Manjaro Testing `5.92.0`
* [ ] Manjaro Unstable `5.93.0`
* [x] OpenSuse Tumbleweed `5.93.0` (fixed `5.93.0-2.1`)
* [ ] Ubuntu 22.04 `5.92.0`
* [x] Arch Linux (fixed in `5.93.0-2`)

**Comment by HugLifeTiZ** [comment]
Strangely, KDE Neon is dragging their feet in getting that fix out. Where should I apply the workaround in PopupView.qml? After the highlighted lines? That's where I put it, but it doesn't seem to make a difference after restarting the Plasma shell.

**Comment by HugLifeTiZ** [comment]
I think Neon is just not going to push out the backport for Frameworks 5.93. I asked in several Matrix channels and got radio silence. They could be busy with migrating from 20.04 to 22.04 anyways. So what's the move?

**Comment by HugLifeTiZ** [comment]
Oh, that's not that far away, luckily.

**Comment by LightDani** [comment]
Just want to inform that it also happens on FerenOS with KDE version 5.24.4 and framework 5.93.0
